wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceService.java

```diff
@@ -928,6 +922,16 @@ private WorkspaceImpl doUpdate(String id, Workspace update)
     }
   }
 
+  private WorkspaceImpl doUpdate(String id, String updateContent)
```

Comment:
I think it's small enough to embed it into `update` method but up to your taste

Reply:
+1 to embed